articles/azure-app-configuration/howto-import-export-data.md

```diff
@@ -479,6 +479,7 @@ From the Azure CLI, follow the steps below. If you don't have the Azure CLI inst
     | `--destination`        | Enter `appservice` to indicate that you're exporting data to App Service.                                                                                          | `appservice`                                                                                                              |
     | `--appservice-account` | Enter the App Service's ARM ID or use the name of the App Service, if it's in the same subscription and resource group as the App Configuration.                   | `/subscriptions/123/resourceGroups/my-as-resource-group/providers/Microsoft.Web/sites/my-app-service` or `my-app-service` |
     | `--label`              | Enter a label to export keys and feature flags with this label. If you don't specify a label, by default, you will only export keys and feature flags with no label. You can enter one label, enter several labels by separating them with `,`, or use `*` to take all of the labels in account. | `prod`                                                                                                                    |
+    | `--export-as-reference` `-r` | Optional. Flag to specify whether key-values are exported to App Service as App Configuration references. This exports references to the latest revision of key-value(s) in App Configuration. [Learn more](../app-service/app-service-configuration-references.md). |
```

Comment:
The `--label` is also optional, but we didn't say it. If we say a parameter is optional, it's important to call out what's the default value when it's not present. The default is missing from our CLI inline help. Can you please add it there too?

> This exports references to the latest revision of key-value(s) in App Configuration.

The "latest version" can be interpreted in many ways, and some suggest auto-refresh while it's not the case. I would recommend removing this sentence. BTW, we also mention the "latest version" in the portal (when hovering on the question mark icon). Can you please remove that too?

Reply:
BTW, it will be great to show an example in this document for key-values exported as references.

Reply:
The reason for adding "latest version" is due to the fact that, for instance, in portal, there is the option to view a version of a key-value from a given date. This was added to clarify that, whenever the referenced key-value is updated in the configuration store, the most up to date version is referenced the next time the application fetches its configuration. Is there any way you would suggest this could be communicated more clearly? Or could this be a source of confusion for users?

Reply:
There are two problems here
1. The "version" is an undefined concept in App Config. Many customers use label to version their key-values. I don't think that's what you meant here.
2. You may have changed a key-value in App Config, but the App Service may still be using the value before it's changed. There is no guarantee when it will fetch the "latest" value. Using the term "latest version" could be interpreted as that App Service will always fetch the "latest" value whenever you make a change in App Config.